FTBS with eoan

running
```
crossbuilder-u eoan -a amd64
```
fails with
```
zigbeenetwork.cpp: In member function ‘void ZigbeeNetwork::setSecurityConfiguration(const ZigbeeSecurityConfiguration&)’:
zigbeenetwork.cpp:107:31: error: implicitly-declared ‘ZigbeeSecurityConfiguration& ZigbeeSecurityConfiguration::operator=(const ZigbeeSecurityConfiguration&)’ is deprecated [-Werror=deprecated-copy]
  107 |     m_securityConfiguration = securityConfiguration;
      |                               ^~~~~~~~~~~~~~~~~~~~~
In file included from zigbeenetwork.h:8,
                 from zigbeenetwork.cpp:2:
zigbeesecurityconfiguration.h:16:5: note: because ‘ZigbeeSecurityConfiguration’ has user-provided ‘ZigbeeSecurityConfiguration::ZigbeeSecurityConfiguration(const ZigbeeSecurityConfiguration&)’
   16 |     ZigbeeSecurityConfiguration(const ZigbeeSecurityConfiguration &other);
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~
cc1plus: all warnings being treated as errors
make[2]: *** [Makefile:598: zigbeenetwork.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[2]: Leaving directory '/home/ubuntu/nymea-zigbee/libnymea-zigbee'
make[1]: *** [Makefile:47: sub-libnymea-zigbee-make_first-ordered] Error 2
make[1]: Leaving directory '/home/ubuntu/nymea-zigbee'
dh_auto_build: make -j5 returned exit code 2
make: *** [debian/rules:26: build] Error 255
```

--- a/libnymea-zigbee/zigbeesecurityconfiguration.cpp
+++ b/libnymea-zigbee/zigbeesecurityconfiguration.cpp
@@ -8,6 +8,7 @@ ZigbeeSecurityConfiguration::ZigbeeSecurityConfiguration()
 ZigbeeSecurityConfiguration::ZigbeeSecurityConfiguration(const ZigbeeSecurityConfiguration &other)
 {
     m_networkKey = other.networkKey();
+    m_zigbeeSecurityMode = other.zigbeeSecurityMode();
     m_globalTrustCenterLinkKey = other.networkKey();
 }
 
@@ -51,6 +52,7 @@ void ZigbeeSecurityConfiguration::clear()
 ZigbeeSecurityConfiguration &ZigbeeSecurityConfiguration::operator=(const ZigbeeSecurityConfiguration &other)
 {
     m_networkKey = other.networkKey();
+    m_zigbeeSecurityMode = other.zigbeeSecurityMode();
     m_globalTrustCenterLinkKey = other.globalTrustCenterLinkKey();
     return *this;
 }

--- a/libnymea-zigbee/zigbeesecurityconfiguration.cpp
+++ b/libnymea-zigbee/zigbeesecurityconfiguration.cpp
@@ -48,6 +48,13 @@ void ZigbeeSecurityConfiguration::clear()
     m_globalTrustCenterLinkKey = "5A6967426565416C6C69616E63653039";
 }
 
+ZigbeeSecurityConfiguration &ZigbeeSecurityConfiguration::operator=(const ZigbeeSecurityConfiguration &other)
+{
+    m_networkKey = other.networkKey();
+    m_globalTrustCenterLinkKey = other.globalTrustCenterLinkKey();
+    return *this;
+}
+
 bool ZigbeeSecurityConfiguration::operator==(const ZigbeeSecurityConfiguration &other) const
 {
     return m_networkKey == other.networkKey() && m_globalTrustCenterLinkKey == other.globalTrustCenterLinkKey() && m_zigbeeSecurityMode == other.zigbeeSecurityMode();

--- a/libnymea-zigbee/zigbeesecurityconfiguration.h
+++ b/libnymea-zigbee/zigbeesecurityconfiguration.h
@@ -26,6 +26,7 @@ public:
 
     void clear();
 
+    ZigbeeSecurityConfiguration &operator=(const ZigbeeSecurityConfiguration &other);
     bool operator==(const ZigbeeSecurityConfiguration &other) const;
     bool operator!=(const ZigbeeSecurityConfiguration &other) const;
 

